toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, Optional
+
+import httpx
+import jwt
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+
+INTERNAL_JWT_SECRET = os.getenv("INTERNAL_JWT_SECRET", "a-very-secret-secret")
```

Comment:
I think the token should be passed when initializating the Client instead of forcing all services to have it as environment variable :)

Reply:
also when you move it, let's avoid defaults. Ideally it should throw an error.

This is because default might mean that if we ever forget to set this value, we will basically have our secret exposed/known on GH :)